notebooks/lesson05_random-forest-deep-dive.ipynb

```diff
@@ -0,0 +1,1577 @@
+{
```

Comment:
<p>Is it possible to store the Viz as an image and display it here so we can see the output? it would be nice if it was persistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>good catch! i decided to drop graphviz and use sklearn's plotting API</p>